subprojects/core/src/integTest/groovy/org/gradle/api/ProfilingIntegrationTest.groovy

```diff
@@ -45,5 +45,6 @@ allprojects {
         document.text().contains("build fooTask")
         document.text().contains("-x barTask")
         output.contains("See the profiling report at:")
+        output.contains("A fine-grained performance profile is available: use the --scan option.")
```

Comment:
Is the build scan profile report really more fine-grained? If yes, in what way? The title of the [coverage on the profiling report in the performance guide](https://guides.gradle.org/performance/#low_level_profiling) somewhat implies that it has more fine-grained information. At least that's what I read out of "low level".

Reply:
Well, not only do you have task timing, but execution order, build cache usage, information about the Gradle Daemon, network, and timings for individual Gradle scripts. It really is _wayyyy_ better ;)

Reply:
Maybe we should use the attribute "detailed" or "feature-richer" instead then. When the message says "fine-grained" then I automatically think "I can drill into the performance numbers more deeply". One of the killer features here is the timings for individual Gradle scripts. Ignore my comment if you don't come to the same conclusion here.